src/platform/utilities/tests/feature-toggles/helpers.unit.spec.js

```diff
@@ -0,0 +1,83 @@
+import { expect } from 'chai';
+
+const {
+  generateToken,
+  getFlipperId,
+} = require('../../feature-toggles/helpers');
+
+describe('feature toogles', () => {
+  let oldCookie;
+
+  before(() => {
+    oldCookie = document.cookie;
+    document.cookie = '';
+  });
+
+  describe('generateToken', () => {
+    it('should return a string', () => {
+      document.cookie = '';
+      expect(generateToken()).to.be.a('string');
+    });
+
+    it('should return a not-null value', () => {
+      document.cookie = '';
+      expect(generateToken()).to.not.be.null;
+    });
+
+    it('should return a not-empty value', () => {
+      document.cookie = '';
+      expect(generateToken()).to.not.be.empty;
+    });
+  });
+
+  describe('getFlipperId', () => {
+    it('should set the FLIPPER_ID cookie to a string', () => {
+      document.cookie = '';
+      expect(getFlipperId()).to.be.a('string');
+    });
+
+    it('should set the FLIPPER_ID cookie to not be null', () => {
+      document.cookie = '';
+      expect(getFlipperId()).to.not.be.null;
+    });
+
+    it('should set the FLIPPER_ID cookie to not be empty', () => {
+      document.cookie = '';
+      expect(getFlipperId()).to.not.be.empty;
+    });
```

Comment:
Can combine these to check that it's a non-empty string.

Also, these say `should set the FLIPPER_ID cookie`, but it's only testing the return value. Below, you check the cookie itself.